server/routes/server.route.js

```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+import validate from 'express-validation';
+import paramValidation from '../../config/param-validation';
+import serverCtrl from '../controllers/server.controller';
+
+const router = express.Router(); // esling-disable-line new-cap
+
+router.route('/')
+  /** GET /api/servers - Get list of servers */
+  .get(serverCtrl.list)
+
+  /** POST /api/servers - Create new server */
+  .post(validate(paramValidation.createServer), serverCtrl.create);
```

Comment:
On devrait proteger ces endpoints par de l'authentification OAuth, pour le moment. Plus tard on va créer un système d'autentification server-to-server à part.